src/features/user/Settings/EditProfile/AddressManagement/UpdateAddressType.tsx

```diff
@@ -33,25 +37,24 @@ const UpdateAddressType = ({
 }: Props) => {
   const tAddressManagement = useTranslations('EditProfile.addressManagement');
   const tCommon = useTranslations('Common');
-  const { contextLoaded, user, token, logoutUser } = useUserProps();
-  const { tenantConfig } = useTenant();
+  const { contextLoaded, user, token } = useUserProps();
+  const { putApiAuthenticated } = useApi();
   const { setErrors } = useContext(ErrorHandlingContext);
   const [isUploadingData, setIsUploadingData] = useState(false);
 
-  const updateAddress = async (addressType: 'primary' | 'mailing') => {
+  const updateAddress = async (addressType: AddressType) => {
     if (!contextLoaded || !user || !token) return;
     setIsUploadingData(true);
-    const bodyToSend = {
+    const bodyToSend: AddressTypeApiPayload = {
       type: addressType,
```

Comment:
Rename bodyToSend to something more relevant, maintaining consistency with the approach elsewhere.